cmd/quasarnoded/main.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/quasarlabs/quasarnode/cmd/quasarnoded/cmd"
 )
 
+var ENV_PREXIX = "QUASARNODED"
```

Comment:
Could yoy please re-check this ENV variable usage that it doe not break anything.?

Reply:
This env prefix is used for : 
// SetEnvPrefix defines a prefix that ENVIRONMENT variables will use.
// E.g. if your prefix is "spf", the env registry will look for env
// variables that start with "SPF_".

Reply:
We can leave it empty if needed. But it does not do much.